docker/Dockerfile

```diff
@@ -31,6 +31,8 @@ COPY config/webserver_config.py ${AIRFLOW_USER_HOME}/webserver_config.py
 RUN chown -R airflow: ${AIRFLOW_USER_HOME}
 RUN chmod +x /entrypoint.sh
 
+RUN yum -y install socat
```

Comment:
Is this going to leave a bunch of cruft in the yum cache? Maybe `yum -y install socat && yum clean all` if there's all the leftover RPM files in there after the build is attempted?

Reply:
Updated.